source/jormungandr/jormungandr/interfaces/v1/GeoStatus.py

```diff
@@ -43,8 +46,14 @@
 }
 
 
-class GeoStatus(Resource):
-    @marshal_with(geo_status)
+class GeoStatus(StatedResource):
+    def __init__(self, *args, **kwargs):
+        super(GeoStatus, self).__init__(output_type_serializer=GeoStatusSerializer, *args, **kwargs)
+
+    @get_serializer(serpy=GeoStatusSerializer, marshall=geo_status)
     def get(self, region):
         instance = i_manager.get_instances(region)[0]
         return {'geo_status': instance.autocomplete.geo_status(instance)}, 200
+
+    def options(self, **kwargs):
```

Comment:
(poke @antoine-de ) Maybe we should hide this endpoint: add `hide=True` to https://github.com/CanalTP/navitia/blob/dev/source/jormungandr/jormungandr/modules/v1_routing/v1_routing.py#L239
Not sure about removing that options here, though.